modules/library/referencing/src/test/java/org/geotools/referencing/factory/URN_ParserTest.java

```diff
@@ -19,51 +19,46 @@
 import org.opengis.referencing.NoSuchAuthorityCodeException;
 
 import org.junit.*;
-import static org.junit.Assert.*;
 
+import static org.junit.Assert.*;
 
 /**
  * Tests the {@link URN_Parser} class.
  *
- *
- *
- * @source $URL$
- * @version $Id$
  * @author Martin Desruisseaux
+ * @version $Id$
+ * @source $URL$
  */
 public final class URN_ParserTest {
 
     /**
      * Parses a valid URN.
      */
-    @Test
-    public void testParse() throws NoSuchAuthorityCodeException {
+    @Test public void testParse() throws NoSuchAuthorityCodeException {
         final URN_Parser parser = URN_Parser.buildParser("urn:ogc:def:CRS:EPSG:6.11.2:4326");
-        assertEquals("crs",       parser.type.name);
-        assertEquals("EPSG",      parser.authority);
-        assertEquals("6.11.2",    parser.version.toString());
-        assertEquals("4326",      parser.code);
+        assertEquals("crs", parser.type.name);
+        assertEquals("EPSG", parser.authority);
+        assertEquals("6.11.2", parser.version.toString());
+        assertEquals("4326", parser.code);
         assertEquals("EPSG:4326", parser.getAuthorityCode());
     }
 
     /**
      * Parses a valid URN without version.
      */
-    @Test
-    public void testParseWithoutVersion() throws NoSuchAuthorityCodeException {
+    @Test public void testParseWithoutVersion() throws NoSuchAuthorityCodeException {
         final URN_Parser parser = URN_Parser.buildParser("urn:ogc:def:CRS:EPSG:4326");
-        assertEquals("crs",       parser.type.name);
-        assertEquals("EPSG",      parser.authority);
-        assertNull  (             parser.version);
-        assertEquals("4326",      parser.code);
+        assertEquals("crs", parser.type.name);
+        assertEquals("EPSG", parser.authority);
+        assertNull(parser.version);
+        assertEquals("4326", parser.code);
         assertEquals("EPSG:4326", parser.getAuthorityCode());
     }
 
     /**
      * Parses an invalid URN.
      */
-    @Test
-    public void testInvalidParse() {
+    @Test public void testInvalidParse() {
```

Comment:
I imported the geotools formatter.xml before starting, maybe it was still using the geogig formatter.